src/utilities/strace/ServerTraceManager.h

```diff
@@ -0,0 +1,174 @@
+/*
+ *	PROGRAM:	Server Access Methods
+ *	MODULE:		ServerTraceManager.h
+ *	DESCRIPTION:	Trace API manager
+ *
+ *  The contents of this file are subject to the Initial
+ *  Developer's Public License Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *  http://www.ibphoenix.com/main.nfs?a=ibphoenix&page=ibp_idpl.
+ *
+ *  Software distributed under the License is distributed AS IS,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing rights
+ *  and limitations under the License.
+ *
+ *  The Original Code was created by Artyom Abakumov
+ *  <artyom.abakumov (at) red-soft.ru> for Red Soft Corporation.
```

Comment:
Really ???

Reply:
Sorry, I didn't mean something like that